artifacts/rhdh-plugins-reference/argocd/argocd-plugin-readme.adoc

```diff
@@ -1,322 +1,268 @@
-        - name: argoInstance1
-          url: https://argoInstance1.com
-          token: ${ARGOCD_AUTH_TOKEN} # Token to use to instance 1
-----
---
-
-=== Permissions in ArgoCD plugin
+[NOTE]
+====
+If you do not set these annotations, the plugin defaults to the first ArgoCD instance configured in `app.config.yaml`.
+====
 
-Setting permissions for ArgoCD user account not only reduces the scope but also reduces the functionality of the backend. If you scope the permissions for read-only, then actions including creating, deleting, and resyncing the application will not be available. 
+== For Users
 
-The error handling in ArgoCD is designed to alert you when proper permissions are not in place.
+.Prerequisites
 
-=== Self-signed certificates in ArgoCD plugin
+.Procedures
 
-By default, the ArgoCD server generates a self-signed certificate. Once you install the ArgoCD plugin, the deployment of argocd-server can be patched to be insecure using the following command:
+* Use the following configuration to install the ArgoCD plugin into Red Hat Developer Hub or Janus IDP through Helm:
 
-[source,console]
++
+[source,yaml]
 ----
-kubectl patch deployment argocd-server --type "json" -p '[{"op":"add","path":"/spec/template/spec/containers/0/command/-","value":"--insecure"}]'
+global:
+  dynamic:
+    includes:
+      - dynamic-plugins.default.yaml
+    plugins:
+      - package: ./dynamic-plugins/dist/roadiehq-backstage-plugin-argo-cd-backend-dynamic
+        disabled: false
+        pluginConfig:
+          argocd:
+            username: "${ARGOCD_USERNAME}"
+            password: "${ARGOCD_PASSWORD}"
+            appLocatorMethods:
+              - type: 'config'
+                instances:
+                  - name: argoInstance1
+                    url: "${ARGOCD_INSTANCE1_URL}"
+                    token: "${ARGOCD_AUTH_TOKEN}"
+      - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-argocd
+        disabled: false
+        pluginConfig:
+          dynamicPlugins:
+            frontend:
+              janus-idp.backstage-plugin-argocd:
+                mountPoints:
+                  - mountPoint: entity.page.overview/cards
+                    importName: ArgocdDeploymentSummary
+                    config:
+                      layout:
+                        gridColumnEnd:
+                          lg: "span 8"
+                          xs: "span 12"
+                      if:
+                        allOf:
+                          - isArgocdAvailable
+                  - mountPoint: entity.page.cd/cards
+                    importName: ArgocdDeploymentLifecycle
+                    config:
+                      layout:
+                        gridColumn: '1 / -1'
+                      if:
+                        allOf:
+                          - isArgocdConfigured
 ----
 
-Also, you can use Helm to install the ArgoCD plugin and be insecure by default:
+=== Loading the Plugin in Backstage Showcase Application
```

Comment:
Can we add this link to the backstage-showcase repo somewhere here https://github.com/janus-idp/backstage-showcase ?

Reply:
I was thinking more about this section, this section is about running the backstage-showcase repo locally by some developers, I dont think we need this section here in RHDH at all, we can remove this section.